rad/src/operators/mixed.rs

```diff
@@ -21,12 +21,12 @@ fn test_as_float() {
     use rmpv::Value;
 
     let radon_float = RadonFloat::from(std::f64::consts::PI);
-    let radon_mixed_error = RadonMixed::from(Value::from(String::from("Hello world!")));
+    //let radon_mixed_error = RadonMixed::from(Value::from(String::from("Hello world!")));
     let radon_mixed = RadonMixed::from(Value::from(std::f64::consts::PI));
 
     assert_eq!(to_float(radon_mixed).unwrap(), radon_float);
-    assert_eq!(
+    /*assert_eq!(
```

Comment:
Don't comment out tests

```suggestion
    assert_eq!(
```